Kernel/System/VirtualFS.pm

```diff
@@ -78,8 +78,10 @@ read a file from virtual file system
 
 returns
 
+    Note that the content value is returned as a reference to a string.
```

Comment:
That comment line is indented. Therefore it is rendered as code. So it should be preceded by a '#'. Or remove the indention for having that comment as regular POD text.

Reply:
Done 👍🏻